### ISSUE TITLE
Feeding a bigger size of the input event than 18.

### ISSUE THREAD

**Issue by terryyz** [atcbosselut/comet-commonsense#25]
Just want to make sure. Are we meant to change the original setting values 17, 35 for ATOMIC to bigger values in order to feed with a longer sentence than 18 words?

Or can we feed with a long sentence?

**Comment by terryyz** [comment]
`XMB = torch.cat((torch.LongTensor(prefix).view( 1, len(prefix)), torch.LongTensor([text_encoder.encoder["<{}>".format(category)]]).view(1,1)), dim =1) ` will help to replace the XMB initialisation.

**Comment by atcbosselut** [comment]
You'd have to change the setting and retrain the model. Training a model with the original settings and then giving sequences longer than 18 words will lead to unstable behavior.
